src/main/resources/com/cloudbees/hudson/plugins/folder/Messages_de.properties

```diff
@@ -0,0 +1,13 @@
+AbstractFolder.BeingDeleted={0} wird gerade gel\u00F6scht
```

Comment:
I would have to look up the specific parent / core versions, but using recent deps/tools I think you can switch to UTF-8. Check https://github.com/jenkinsci/jenkins/pull/6719 https://github.com/jenkinsci/jenkins-test-harness/pull/448